lib/librato/rails/subscribers/mail.rb

```diff
@@ -4,15 +4,19 @@ module Subscribers
 
       # ActionMailer
 
-      ActiveSupport::Notifications.subscribe 'deliver.action_mailer' do |*args|
-        # payload[:mailer] => 'UserMailer'
-        collector.increment "rails.mail.sent"
-      end
+      %w{sent received}.each do |metric|
+
+        ActiveSupport::Notifications.subscribe "deliver.action_mailer" do |*args|
+
+          event = ActiveSupport::Notifications::Event.new(*args)
+          tags = { mailer: event.payload[:mailer] }
+
+          collector.increment "rails.mail.#{metric}", tags: tags
+
+        end # end subscribe
 
-      ActiveSupport::Notifications.subscribe 'receive.action_mailer' do |*args|
```

Comment:
I think this breaks managing received mail? It looks like are losing the subscription to `receive.action_mailer` here.